spec/seagate_ilm.spec

```diff
@@ -0,0 +1,120 @@
+Summary: A lock manager based on Seagate In-Drive-Mutex (IDM)
+Name: seagate_ilm
+Epoch: 1
+Version: 1
+Release: 1%{?dist}
```

Comment:
You could see that the propeller repo has a file VERSION, this file can be used to read out the version number.  And we can study sanlock to dynamically read out the release version by using the command:

~~~
ifeq ($(shell git describe --exact-match 2>/dev/null),)
# sanlock-3.7.0-5-g11fb098 -> 5.g11fb098
release := $(shell git describe --tags | awk -F- '{print $$(NF-1) "." $$(NF)}')
else
release := 0
endif
~~~

And you could see that sanlock uses a spec.in file, and Makefile can dynamically generate the sanlock.spec file based on sanlock.spec.in.  The benefit for this approach is we can only maintain the VERSION file and add new tag with "git tag" command, then the spec file can dynamically update its version number and release number based on the latest VERSION file ang git tag.

Reply:
I'll take a look at this.